.gitlab-ci.yml

```diff
@@ -512,6 +532,26 @@ release_image:
     - tags
     - triggers
 
+release_trace-agent:
+  stage: release
+  image: docker:18.06.0
+  dependencies:
+    - deps_deb
+    - build_deb
+  variables:
+    DOCKER_DRIVER: overlay
+  services:
+    - docker:dind
+  
+  script:
+    - cd $CI_PROJECT_DIR
+    - export VERSION=`cat version.txt`
+    - export PUSH_LATEST=true
+    - ./omnibus/package-scripts/publish_trace_agent.sh $VERSION $STS_DOCKER_RELEASE_REPO $CI_PROJECT_DIR/outcomes/binary/trace-agent $PUSH_LATEST
```

Comment:
don't use `$STS_DOCKER_RELEASE_REPO` instead define a variable at the top like the other `STS_DOCKER_RELEASE_REPO_TRACE: stackstate-trace-agent` and use it

Reply:
added release repo